spec/std/isa/inst/B/rev8.yaml

```diff
@@ -20,21 +20,31 @@ description: |
 definedBy:
   anyOf: [Zbb, Zbkb]
 assembly: xd, xs1
-encoding:
+format:
   RV32:
```

Comment:
The (new) "format" stanza does not support "RV32"/"RV64" like "encoding" does. @dhower-qc was that intentional?